src/test/java/ioanandroflorea/tests/Tema8/pageObjects/RefreshPage.java

```diff
@@ -0,0 +1,32 @@
+package ioanandroflorea.tests.Tema8.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class RefreshPage {
+    ChromeDriver driver;
+    WebDriverWait wait;
+
+    public RefreshPage(ChromeDriver driver){
+        this.driver = driver;
+        wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        PageFactory.initElements(driver,this);
+    }
+    @FindBy(id ="refreshdate")
+    WebElement refreshButton;
+    public String getCurrentTimestamp(){
+        return refreshButton.getText();
+    }
+    public String getRefreshedTimestamp(){
```

Comment:
This method does the same thing as getCurrentTimestamp(). No need to duplicate it. Keep only one of them and use it whenever you need it and remove the other one. Thanks!